src/presentation/http/router/userList.test.ts

```diff
@@ -0,0 +1,63 @@
+import userSessions from '@tests/test-data/user-sessions.json';
+import { describe, test, expect, beforeAll } from 'vitest';
+
+
+/**
+ * Access token that will be used for Auhorization header
+ */
+let accessToken = '';
+
+describe('UserList API', () => {
+  beforeAll(() => {
+    /**
+     * userId for authorization
+     */
+    const userId = userSessions[0]['user_id'];
+
+    accessToken = global.auth(userId);
+  });
+
+  describe('GET /users?page', () => {
+    test.each([
+      { portionSize : 3,
+        pageNumber: 1 },
+
+      { portionSize: 0,
+        pageNumber : 2 },
+    ])
+    ('Returns a userList with specified lengths with status 200', async ({ portionSize, pageNumber }) => {
+      const expectedStatus = 200;
+
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/users?page=${pageNumber}`,
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      expect(response?.json().items).toHaveLength(portionSize);
```

Comment:
maybe we should have special variable for `response?.json()` as in AAA model

Reply:
Last time neSpecc pointed out that a special variable was not needed